.github/CONTRIBUTING.md

```diff
@@ -22,24 +23,27 @@ is as follows:
 
 If you're not used to this workflow with git, you can start with some [basic docs from GitHub](https://help.github.com/articles/fork-a-repo/) or even their [excellent interactive tutorial](https://try.github.io/).
```

Comment:
Is this part of the template?

Reply:
Yes it is: https://github.com/nf-core/tools/blob/master/nf_core/pipeline-template/%7B%7Bcookiecutter.name_noslash%7D%7D/.github/CONTRIBUTING.md

Reply:
Must have missed that at some point...
So which tutorial are we mentioning, because this URL is now `Resources to learn Git` with tutorials and docs.

Reply:
Good point - should probably refer to the interactive tutorials then? People might be interested in different tutorials there, as they have different background / prior experience?